src/main/java/gov/usgs/earthquake/util/JSONTrackingFile.java

```diff
@@ -0,0 +1,76 @@
+package gov.usgs.earthquake.util;
+
+import java.io.ByteArrayInputStream;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+
+import javax.json.Json;
+import javax.json.JsonObject;
+import javax.json.JsonReader;
+
+import gov.usgs.util.FileUtils;
+
+/**
+ * Represents a tracking file in JSON.
+ */
+public class JSONTrackingFile {
+  private JsonObject json;
```

Comment:
I would get rid of `json` and `trackingFile`.  This can just encapsulate the writing and parsing.

Reply:
Refactored to use only `trackingFileName` as member variable